core-api/src/main/java/com/optimizely/ab/bucketing/Bucketer.java

```diff
@@ -102,6 +109,19 @@ private Variation bucketToVariation(@Nonnull Experiment experiment,
                                         @Nonnull String userId) {
         // "salt" the bucket id using the experiment id
         String combinedBucketId = userId + experiment.getId();
+
+        // If a persistent bucketer instance is present then check it for a persisted variation
```

Comment:
I think we are doing the right thing, but confirming. Whitelisting happens before `bucketToVariation` is called right?